### PR TITLE
Updated Grants icon in nav

### LINKED ISSUE #2961: Nav: Change icon of 'Send Kudos' and 'My Grants' on nav dropdown
### Current Behavior

Icons of 'Send Kudos' and 'My Grants' on nav dropdown do not represent the branding of the feature
screenshot 2018-11-28 01 17 30

### Expected Behavior
Change the icons of 'Send Kudos' and 'My Grants' following the screenshot below:
screenshot 2018-11-28 01 16 26

Sketch files (new icons inside):
https://drive.google.com/open?id=1JLMtaFe3ul7nyWgH4q_1cxCIFOImUUEV

### PR DESCRIPTION
**Pull request by usmanmuhd**
Fixes #2961

### CODE REVIEW

**Comment by usmanmuhd** [comment]
@thelostone-mc Could you take a look at this now?

File: app/dashboard/templates/shared/nav_auth.html [hunk 1]
```diff
@@ -41,7 +41,7 @@
           {% trans "Fund Issue" %}
         </a>
         <a class="dropdown-item" href="{% url "grants:profile" %}">
-          <i class="fas fa-money-check"></i>
+          <i class="fas"><img src="{% static "v2/images/grants/IconMyGrants.svg" %}"></i>
```
Comment (thelostone-mc) [inline app/dashboard/templates/shared/nav_auth.html]: do we need the `fas` class? @usmanmuhd

Comment (thelostone-mc) [reply app/dashboard/templates/shared/nav_auth.html]: same comment for kudos one. I merged that [PR](https://github.com/gitcoinco/web/pull/2971/files#r237012001).
Could you open a new one thought?

Comment (usmanmuhd) [reply app/dashboard/templates/shared/nav_auth.html]: It is required to keep the icon aligned in the same way as the other icons.

**Pull request merged**